zebra-chain/src/sapling/commitment/pedersen_hashes.rs

```diff
@@ -91,6 +91,8 @@ pub fn pedersen_hash_to_point(domain: [u8; 8], M: &BitVec<Lsb0, u8>) -> jubjub::
 
 /// Pedersen Hash Function
 ///
+/// This is technically returning 255 (l_MerkleSapling) bits, not 256.
+///
```

Comment:
should we new-type this to ensure we don't mix them up in the future?

Reply:
No I don't think it matters because it feeds into a scalar (`Fq`) anyway so it gets washed away, this comment is more about how it matches the mathematical function in the spec, maybe I should just remove it